app/src/main/res/layout/simple_video_layout.xml

```diff
@@ -12,52 +12,10 @@
   android:foreground="?attr/selectableItemBackgroundBorderless">
```

Comment:
We already have `homeIllustration` view in the patients tab screen. So, we can remove this layout.